src/muya/lib/contentState/pasteCtrl.js

```diff
@@ -116,7 +116,7 @@ const pasteCtrl = ContentState => {
           link.innerHTML = sanitize(title, PREVIEW_DOMPURIFY_CONFIG, true)
         } else {
           const span = document.createElement('span')
-          span.innerHTML = text
+          span.innerHTML = sanitize(title, PREVIEW_DOMPURIFY_CONFIG, true)
```

Comment:
One question: shouldn`t the text where the title is? for example `sanitize(text, PREVIEW_DOMPURIFY_CONFIG, true)`?